tests/test_committees.py

```diff
@@ -625,8 +624,7 @@ def test_is_active(self):
                 CommitteeHistoryProfileView,
                 committee_id=self.committees[2].committee_id,
                 cycle=2014,
-                election_full=False,
-                is_active=True,
```

Comment:
So much great work with these tests! It makes sense why you removed is_active because it's not a filter. For the test_is_active test, however, do you think we should change the 2 assert statements to the is_active value like this:
 assert results[0].get("is_active") == False

Reply:
That sounds like a good idea, I'll make the change!